tests/PlancakeEmailParserTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+declare(strict_types = 1);
+use PHPUnit\Framework\TestCase;
+
+class PlancakeEmailParserTest extends TestCase {
+
+  public function testSubject() {
+    foreach (glob(__DIR__ . '/emails/*.txt') as $testFile) {
```

Comment:
Brace placement and indentation needs to follow the existing conventions of this codebase (which generally follow PSR-2).